pages/getting-started/first-steps-with-docker.mdx

```diff
@@ -6,455 +6,191 @@ description: Starting with Docker? Peruse our detailed guide outlining the first
-processes other than Docker. Any changes made to these directories or files
-locally will be reflected inside the Docker container and vice-versa. Also, a
-bind mount will overwrite the content of the Docker container.
-
-For example, if I have a `Data` directory on my `C:` disk, and I want to access
-it from inside the container at `/usr/lib/memgraph/data`, I would run Docker
-with the following `-v` flag.
+2. Start the container by ID or name:
 
 ```
-docker run -p 7687:7687 -p 7444:7444 -v "C:/data":/usr/lib/memgraph/data memgraph/memgraph-mage
+docker start <CONTAINER_ID>
+# or
+docker start <CONTAINER_NAME>
 ```
 
-You can use bind mounts to transfer durability files such as snapshot or WAL
-files inside the container to restore data, or CSV files you will use to import
-data with `CSV LOAD` clause. 
-
-Bind mounts do not handle data permissions which could cause issues with
-permissions. 
-
-### Stop container
+### Delete a container
 
-<Callout type="info">
-Stopping the container isn't the same thing as [deleting it](#delete-container).
+<Callout type="warning"> 
+  Deleting a container is permanent. Make sure to back up any important data
+  before proceeding.
 </Callout>
 
-Database instances are stopped by stopping the Docker container with the command
-`docker stop`. To stop a container you need [to know the container's
-ID](#how-to-retrieve-a-docker-container-id) or [name](#container-name).
+Before deleting a container, ensure it’s stopped. Then use the docker rm command
+with the container’s [ID](#get-docker-containers-id) or name:
 
-You can list all the containers you want to stop in one `docker stop` command:
-
-```
-docker stop CONTAINER1_ID CONTAINER2_ID CONTAINER3_NAME
 ```
-
-### Start container
-
-If you want to start a stopped container, list them using the following command: 
-
-```
-docker ps -a
-```
-
-Then start the container with: 
-```
-docker start <CONTAINER_ID> 
+docker rm <CONTAINER_ID>
+# or
+docker rm <CONTAINER_NAME>
 ```
 
-or 
+To remove multiple containers at once:
 
 ```
-docker start <CONTAINER_NAME> 
+docker rm CONTAINER1_ID CONTAINER2_ID CONTAINER3_NAME
 ```
 
-### Delete container
+### `docker run` vs `docker start`
 
-<Callout type="warning">
-Deleting a container will permanently remove it from your system. Ensure you have backed up any important data before proceeding.
-</Callout>
-
-To remove a Docker container, you first need to ensure that it's stopped. Once
-it's stopped, you can use the `docker rm` command. Like stopping a container,
-you'll need [to know the container's ID](#retrieve-a-docker-container-id) or
-[name](#container-name) to delete it.
+While they might sound similar, `docker run` and `docker start` serve very
+different purposes:
 
-Here's how you can delete a single container:
+`docker run` — **create & start a new container**
+- Initializes a new container from a specified image.
+- Pulls the image from Docker Hub if it’s not available locally.
```

Comment:
I would add some kind of warning here or below that latest pulls the latest locally available image and to make sure to first run pull then run to ensure you have the latest image